python/paddle/fluid/dygraph/dygraph_to_static/ifelse_transformer.py

```diff
@@ -587,57 +580,108 @@ def _vars_with_store(ids_dict):
                 vars.append(k)
         return vars
 
-    def _candidate_vars(child_dict, parent_dict):
+    def _modified_vars(child_dict, parent_dict):
         return set([
             var for var in _vars_with_store(child_dict) if var in parent_dict
         ])
 
-    # 1. the var in parent_ids is modified in if/else node.
-    if_candidate_vars = _candidate_vars(if_vars_dict, parent_vars_dict)
-    else_candidate_vars = _candidate_vars(else_vars_dict, parent_vars_dict)
-
-    # 2. new var is both created in if and else node.
-    if_new_vars = set([
+    def _vars_before_store(ids_dict):
```

Comment:
maybe `_vars_loaded_before_store` is better?

Reply:
Thanks, done